docs/home/howto/MAINTENANCE.md

```diff
@@ -33,17 +33,17 @@ We can also check state of Kubernetes nodes using the command:
 
 ```shell
 root@primary01:~# kubectl get nodes --kubeconfig=/etc/kubernetes/admin.conf
-NAME                                         STATUS   ROLES    AGE   VERSION
-primary01                                    Ready    master   24h   v1.17.7
-node01                                       Ready    <none>   23h   v1.17.7
-node02                                       Ready    <none>   23h   v1.17.7
+NAME                                         STATUS   ROLES                  AGE   VERSION
+primary01                                    Ready    control-plane,master   24h   v1.21.7
```

Comment:
1.22 is the version we should have after this release:) Or maybe put a generic x.xx.x something so we don't need to touch this again later?